src/SearchResults.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
+
+const CountNumberOfNights = (checkInDate, checkOutDate) => {
+  let a = moment(checkInDate);
+  let b = moment(checkOutDate);
+  return b.diff(a, "days");
+};
+
+const SearchResults = (props) => {
+  const [isActive, setIsActive] = useState(false);
+  const [selectedID, setSelectedId] = useState(null);
+
+  function highlightSelectedRow(id) {
+    setIsActive(!isActive);
+    setSelectedId(id);
+  }
```

Comment:
I think you're mixing the logic of highlighting the row when it is clicked (i.e. task 15) and clicking on the Show profile button (i.e. tasks 20 and 21).

Clicking on the row should highlight it without making any changes to the customer profile. Additionally, the user should be able to highlight multiple rows at the same time.

Trying clicking on different multiple rows in https://strong-meerkat-359514.netlify.app/ to experience how it should look like. Let me know if you can't spot the differences and I'll try to elaborate more.

Reply:
Hi @sherif98 , may I ask for help with this issue, please?